src/components/emptyChat/index.ts

```diff
@@ -0,0 +1,23 @@
+import emptyChat from './emptyChat.hbs';
+import './emptyChat.scss'
+import Component from '../../utils/Component';
+
+type EmptyChatProps = {
+    title: string,
+    subtitle: string,
+    button: object,
+    attr?: object,
+    events?: object
```

Comment:
я думаю, использование object и string не сильно отличается от any. Я думаю можно детальнее типизировать, например так
```suggestion
    button: object,
    attr?: 'height' | 'width',
    events?: Record<'click' | 'blur', () => void>
```

Это пример, т.к. я не знаю твоей задумки, но ты можешь сделать по аналогии. 

Почему это важно и я заостряю на этом внимание?

Основная задача typescript'a - помогать нам на этапе написания кода. Если мы допускаем ошибку например в названии переменной и typescript дает нам подсказку, то он используется эффективно. Если мы используем такие типы как object, string и тд, то туда можно поместить любой объект и любую строку, а значит мы можем ошибиться и ts нам не подскажет. В общем надо стараться максимально детальее все типизировать